src/library/Account/Wrapper.ts

```diff
@@ -6,10 +6,10 @@ import type { WrapperProps } from './types';
 
 export const Wrapper = styled.button<WrapperProps>`
   border: 1px solid var(--border-primary-color);
-  cursor: ${(props) => (props.canClick ? 'pointer' : 'default')};
+  cursor: ${(props) => (props.$canclick ? 'pointer' : 'default')};
```

Comment:
I am not getting any warnings when using camel case names here.

Reply:
I did for this one and some are fine to me too, but for the code consistency and in case they has different actions on different users I have edit all the props to lowercase.

Reply:
Can you change to camel case? This seems to be the standard & is what is being used in docs and walkthroughs.

Reply:
Okay, I can do it but the warnings might be back again

Reply:
Well I didn't get any, can you share a screenshot of them if you do and we can look at how to resolve them.